apps/src/applab/designMode.js

```diff
@@ -359,12 +359,18 @@ designMode.updateProperty = function(element, name, value) {
     case 'picture':
       originalValue = element.getAttribute('data-canonical-image-url');
       element.setAttribute('data-canonical-image-url', value);
```

Comment:
We should make this same change for 'screen-image' and 'image' so backgrounds and buttons aren't susceptible to the same issue.